library/common/jni/BUILD

```diff
@@ -199,3 +181,24 @@ cc_library(
         "//library/common/jni/import:jni_import_lib",
     ],
 )
+
+cc_library(
+    name = "jni_utility_lib",
+    srcs = [
+        "jni_utility.cc",
+        "jni_version.cc",
+    ],
+    hdrs = [
+        "jni_support.h",
+        "jni_utility.h",
+        "jni_version.h",
+    ],
+    deps = [
+        "//library/common/jni/import:jni_import_lib",
+        "//library/common/types:c_types_lib",
+        "@envoy//source/common/common:assert_lib",
+    ] + select({
+        "@envoy//bazel:android_logger": [":ndk_jni_support"],
```

Comment:
@keith this is probably not the right select for using android - do you know if there's a better opton?

Reply:
We have this internally https://github.com/envoyproxy/envoy/pull/21459 seems useful upstream at this point, wdyt